doc/manual/management_system.rst

```diff
@@ -143,6 +143,77 @@ CCBs are used by experiments to configure applets in the dashboard, for example
 .. autoclass:: artiq.dashboard.applets_ccb.AppletsCCBDock
    :members:
 
+Applet IPC Clients
+******************
+
+Applet IPC clients allow applets to perform actions on the dashboard and master databases. Applets may inherit from the ``artiq.applets.simple.SimpleApplet`` and call the methods defined below through the `ctl` object.
+
+Embedded applets should use `AppletControlIPC` while standalone applets use `AppletControlRPC`. `SimpleApplet` automatically chooses the correct client on initialization. 
+
+.. autoclass:: artiq.applets.simple._AppletControl
+   :members:
+
+
+Applet Entry Area
+*****************
+
+Extensions are provided to enable the use of argument widgets in applets through the `EntryArea` class. 
+
+Below is a simple example code snippet using the `EntryArea` class: ::
+
+   # Create the experiment area
+   entry_area = EntryArea()
+
+   # Create a new widget
+   entry_area.setattr_argument("bl", BooleanValue(True))
+
+   # Get the value of the widget (output: True)
+   print(entry_area.bl) 
+
+   # Set the value
+   entry_area.set_value("bl", False)
+
+   # False
+   print(entry_area.bl)
+
+The `EntryArea` object can then be added to a layout and integrated with the applet GUI. Multiple `EntryArea` objects can be used in a single applet.
+
+.. py:class:: artiq.gui.applets.EntryArea
```

Comment:
Only used it since as mentioned in the notes we currently mock gui/applets.py. Was not a problem to compile, and it is valid sphinx syntax, but can do some additional research and testing to get more confidence

Reply:
Removed the python domain, using `.. class` syntax instead. No difference in the output. `.. autoclass` produces  source code like `.. class` anyways according to [this doc](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html)